src/swapd/swap_state.rs

```diff
@@ -9,6 +9,7 @@ use crate::bus::p2p::Commit;
 use crate::bus::Outcome;
 
 #[derive(Display, Debug, Clone, StrictEncode, StrictDecode)]
+
```

Comment:
I guess this isn't intentional

Reply:
Mmh, let me see. I'm sure I can come up with a reason for adding random whitespace, but in the meantime: ba51dcc5f6b4aae1affe87e51b4b04239666e895